www/pages/docs/server-rendering.md

```diff
@@ -83,9 +86,10 @@ export default class UsersPage extends HTMLElement {
 }
 ```
 
-In the above example, _card.js_ will automatically be bundled for you on the client side!  🙌
-
-> _**Note**: Keep in mind that for these "page" components, you will likely want to _avoid_ Declarative Shadow for rendering the top level (to avoid wrapping static content in `<template>` tags), but definitely use Declarative Shadow DOM within any dependent custom elements of your page._
+A couple of notes:
+- WSCs run only on the server, thus you have full access to any APIs of the runtime, with the ability to perform one time `async` operations for [data loading](/docs/server-rendering/#data-loading) in `connectedCallback`.
+- In the above example, card.js will automatically be bundled for you on the client side!
```

Comment:
Hmm... goes Greenwood actually do this? 🤔 

I feel like yes, but in reality I don't think I've observed it?  Can't say I've directly looked for it since I didn't think it was actually a feature 😅